src/bitmessagemain.py

```diff
@@ -405,12 +406,22 @@ def getApiAddress(self):
         port = BMConfigParser().getint('bitmessagesettings', 'apiport')
         return {'address':address,'port':port}
 
+    
+def emojiLogg():
```

Comment:
Please rename to something else than "emoji", e.g. something with utf8, unicode or encoding.